libs/plpgsql.json

```diff
@@ -0,0 +1,46 @@
+  {"id": "plpgsql.earthdistance", "imports": ["earthdistance"], "name": "earthdistance", "repo": "postgres/postgres/tree/master/contrib/earthdistance", "tags": ["geo-coords"], "tech": ["utils"]},
+  {"id": "plpgsql.file-fdw", "imports": ["file_fdw"], "name": "file_fdw", "repo": "postgres/postgres/tree/master/contrib/file_fdw", "tags": ["remote-control"], "tech": ["utils"]},
+  {"id": "plpgsql.fuzzystrmatch", "imports": ["fuzzystrmatch"], "name": "fuzzystrmatch", "repo": "postgres/postgres/tree/master/contrib/fuzzystrmatch", "tags": ["string-distance"], "tech": ["utils"]},
+  {"id": "plpgsql.hstore", "imports": ["hstore"], "name": "hstore", "repo": "postgres/postgres/tree/master/contrib/hstore", "tags": ["hashmap"], "tech": ["data-structures"]},
+  {"id": "plpgsql.intarray", "imports": ["intarray"], "name": "intarray", "repo": "postgres/postgres/tree/master/contrib/intarray", "tags": ["array"], "tech": ["storage"]},
+  {"id": "plpgsql.isn", "imports": ["isn"], "name": "isn", "repo": "postgres/postgres/tree/master/contrib/isn", "tags": ["numbers"], "tech": ["computational-science"]},
+  {"id": "plpgsql.lo", "imports": ["lo"], "name": "lo", "repo": "postgres/postgres/tree/master/contrib/lo", "tags": ["large-objects"], "tech": ["computational-science"]},
+  {"id": "plpgsql.ltree", "imports": ["ltree"], "name": "ltree", "repo": "postgres/postgres/tree/master/contrib/ltree", "tags": ["ltree"], "tech": ["data-structures"]},
+  {"id": "plpgsql.pageinspect", "imports": ["pageinspect"], "name": "pageinspect", "repo": "postgres/postgres/tree/master/contrib/pageinspect", "tags": ["db-analysis"], "tech": ["eng-tools"]},
+  {"id": "plpgsql.passwordcheck", "imports": ["passwordcheck"], "name": "passwordcheck", "repo": "postgres/postgres/tree/master/contrib/passwordcheck", "tags": ["password-strength"], "tech": ["security"]},
+  {"id": "plpgsql.pg_buffercache", "imports": ["pg_buffercache"], "name": "pg_buffercache", "repo": "postgres/postgres/tree/master/contrib/pg_buffercache", "tags": ["db-analysis"], "tech": ["eng-tools"]},
+  {"id": "plpgsql.pgcrypto", "imports": ["pgcrypto"], "name": "pgcrypto", "repo": "postgres/postgres/tree/master/contrib/pgcrypto", "tags": ["security"], "tech": ["cryptography"]},
+  {"id": "plpgsql.pg-freespacemap", "imports": ["pg_freespacemap"], "name": "pg_freespacemap", "repo": "postgres/postgres/tree/master/contrib/pg_freespacemap", "tags": ["free-space-map"], "tech": ["eng-tools"]},
+  {"id": "plpgsql.pg-prewarm", "imports": ["pg_prewarm"], "name": "pg_prewarm", "repo": "postgres/postgres/tree/master/contrib/pg_prewarm", "tags": ["caching"], "tech": ["faster-code"]},
+  {"id": "plpgsql.pgrowlocks", "imports": ["pgrowlocks"], "name": "pgrowlocks", "repo": "postgres/postgres/tree/master/contrib/pgrowlocks", "tags": ["db-analysis"], "tech": ["eng-tools"]},
```

Comment:
I think it's also diagnostics. eng-tools is more for things like code generation and such. diagnostics for what you use to debug problems.